color_balance/colorimage.py

```diff
@@ -56,7 +45,8 @@ def convert_to_colorimage(cimage, band_indices=None,
     :returns: colorimage (3-band, 8-bit BRG openCV array), mask (1-band, 8-bit
```

Comment:
I gather this is no longer an opencv array- we should update the docs accordingly.